piptools/scripts/sync.py

```diff
@@ -84,6 +84,15 @@ def cli(
     src_files,
 ):
     """Synchronize virtual environment with requirements.txt."""
+    try:
+        fwd_args_idx = src_files.index("--")
+    except ValueError:
+        install_flags = []
+    else:
+        install_flags = list(src_files[fwd_args_idx + 1 :])  # noqa
```

Comment:
Could you update in a separate PR `setup.cfg` to ignore `E203`? See https://github.com/psf/black#slices.

Reply:
Sure, see #1081. I'm guessing that one doesn't need tests or changelog entries.

Reply:
Accordingly, I've removed the one-off noqa marker, so the tests will fail until the #1081 is merged.

Reply:
Merged.